File: app/src/main/java/org/take365/Network/Models/StoryImageImagesModel.java
package org.take365.Network.Models;

import java.io.Serializable;

/**
 * Created by evgeniy on 08.02.16.
 */
public class StoryImageImagesModel implements Serializable {
    public int id;
    public String title;
    public String description;
    public StoryImageThumbModel thumb;
    public StoryImageThumbModel thumbLarge;
    public StoryImageThumbModel image;
    public StoryImageThumbModel imageLarge;
    public String date;
    public long timestamp;
    public long likesCount;
    public boolean isLiked;
}

File: app/src/main/java/org/take365/Network/Models/StoryImageImagesModel.kt
package org.take365.Network.Models

import java.io.Serializable

/**
 * Created by evgeniy on 08.02.16.
 */
data class StoryImageImagesModel(
        var id: Int,
        var title: String,
        var description: String?,
        var thumb: StoryImageThumbModel,
        var thumbLarge: StoryImageThumbModel,
        var image: StoryImageThumbModel,
        var imageLarge: StoryImageThumbModel,
        var date: String,
        var timestamp: Long,
        var likesCount: Long,
        var isLiked: Boolean
) : Serializable
